src/digitalmarketplace/common.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * TODO: Ideally this would be a NodeList.prototype.forEach polyfill
+ * This seems to fail in IE8, requires more investigation.
+ * See: https://github.com/imagitama/nodelist-foreach-polyfill
+ */
```

Comment:
do we still need to support IE8? Have you tried the polyfills at https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#Polyfill ?

Reply:
Alteratively we could do this without polyfill using one one the hacks described in https://css-tricks.com/snippets/javascript/loop-queryselectorall-matches/ ?

Reply:
This file was previously copied from https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/common.js, which does have some support for IE8. I figured I'd keep it in place for maximum compatibility, but I guess since that method is 2+ years old, and we don't have explicit support for IE8 on DMp, I'll swap this out for a polyfill.